_data/blog.yml

```diff
@@ -1,21 +1,11 @@
 posts:
-- title: "Leisure Gridcoin client update v4.0.1.0!"
-  link_primary_url: "https://github.com/gridcoin-community/Gridcoin-Research/releases/tag/4.0.1.0"
-  link_primary_text: "v4.0.1.0 release notes"
+- title: "Leisure Gridcoin client update v4.0.4.0!"
+  link_primary_url: "https://github.com/gridcoin-community/Gridcoin-Research/releases/tag/4.0.4.0"
+  link_primary_text: "v4.0.4.0 release notes"
   link_secondary_url: ""
   link_secondary_text: ""
-- title: "MANDATORY Gridcoin client update v4.0.0.0!"
```

Comment:
Do we just want to remove the posts about the old updates?

Reply:
I think the mandatory still should probably be kept to let people know what the last mandatory version was

Reply:
Mmm I had though it might be best to at least list the V4 series of releases. I'll fix that real quick.

Reply:
Yeah just doing the version 4 updates would probably be fine